front/lib/api/assistant/agent.ts

```diff
@@ -187,18 +189,18 @@ export type AgentActionSuccessEvent = {
   action: AgentActionType;
 };
 
-// Event sent when tokens are streamed as the the agent is generating a message.
-export type AgentGenerationTokensEvent = {
-  type: "agent_generation_tokens";
+// Event sent once the generation is completed.
+export type AgentGenerationSuccessEvent = {
+  type: "agent_generation_success";
   created: number;
   configurationId: string;
   messageId: string;
   text: string;
 };
 
 // Event sent once the message is completed and successful.
-export type AgentGenerationSuccessEvent = {
-  type: "agent_generation_success";
+export type AgentSuccessEvent = {
```

Comment:
NIT: Looking at the comment, should it be "AgentMessageSuccessEvent"? (asking as "once the message is completed and successful" means it is attached to the generation, right?)
 
Or will we use this event also after a successful action and it's the comment that needs to be updated?

Reply:
Happy to call it AgentMessageSuccessEvent :+1: The idea is for this one to be the final success event of the agent meaning that everything is ready and done. Since we have `succeeded` on AgentMessage, let's add Message in the event name here :+1: